word_parser.rb

```diff
@@ -0,0 +1,52 @@
+module WordParser
+  def self.read_file(file_path)
+    File.open(file_path) { |f| f.read }
```

Comment:
I wasn't sure if this closed the file, or if it was necessary to close the file after a read. I did see that if a block was used, the file is automatically closed. That was my logic for doing it this way.